pkg/odo/cli/component/common_push.go

```diff
@@ -75,14 +74,18 @@ func (cpo *CommonPushOptions) createCmpIfNotExistsAndApplyCmpConfig(stdout io.Wr
 				cmpName,
 				err,
 			)
+
 			os.Exit(1)
 		}
 	}
 
 	// Apply config
 	err := component.ApplyConfig(cpo.Context.Client, *cpo.localConfigInfo, stdout, cpo.doesComponentExist)
 	if err != nil {
-		odoutil.LogErrorAndExit(err, "Failed to update config to component deployed")
+		log.Errorf("Failed to update config to component deployed. Error: %v",
+			err,
+		)
+		os.Exit(1)
```

Comment:
we don't use exit in this package. We have helper functions like `LogErrorAndExit` for that

Reply:
@mik-dass Please check here https://github.com/openshift/odo/blob/master/pkg/odo/cli/component/common_push.go#L73 in the same file it is being used, so i have also used it.
in `LogErrorAndExit` it is not printing error in case context given, so either i need to change it or do not pass context here.

Reply:
https://github.com/openshift/odo/blob/master/pkg/odo/util/cmdutils.go#L52

Reply:
> @mik-dass Please check here https://github.com/openshift/odo/blob/master/pkg/odo/cli/component/common_push.go#L73 in the same file it is being used, so i have also used it.
 
Then we need to remove it from there too in a separate PR. Here's a discussion regarding that https://github.com/openshift/odo/pull/2103#issuecomment-532690129

> in LogErrorAndExit it is not printing error in case context given, so either i need to change it or do not pass context here.

I guess we can avoid the context in this case as the current one isn't clear enough.

Reply:
@mik-dass Thanks for the reference, I will update the PR.